lifetimes/estimation.py

```diff
@@ -81,7 +81,7 @@ def _sum(x, tx, recency_T):
 
     @staticmethod
     def _negative_log_likelihood(params, frequency, recency, n, n_custs, penalizer_coef=0):
-        penalizer_term = penalizer_coef * log(params).sum()
+        penalizer_term = penalizer_coef * sum(params)
```

Comment:
params is sometimes a Python list, so no builtin sum. The list has maximum 4 elements, and doing some offline `timeit` tests shows that Python's `sum` is actually really fast